### PR TITLE
feat: (IAC-842) Add V4_CFG_CERTS and conditionally download viya4-orders-cli

### PR DESCRIPTION
**Pull request by jarpat**
### Changes
* Added `V4_CFG_CERTS` to allow the user to define a path to a pre-downloaded certificates file
* The viya4-orders-cli is now conditionally downloaded during ansible execution 
  * If the user defines `V4_CFG_LICENSE`, `V4_CFG_CERTS`,  & `V4_CFG_DEPLOYMENT_ASSETS` the CLI is no longer installed since we already have all the required files on disk.

### Tests

Testing Notes:

* Ran all of these with `DEPLOY:false` since we are only testing the assets.yaml functionality.
* For all these scenarios I had my assets in an external directory (NOT inside `DEPLOY_DIR`) and mounted set my parameters as needed to point to those files. 
* Between each test my `DEPLOY_DIR` was wiped clean so I could ensure if we are using user provided assets, they are copied/extracted to the correct location inside the `DEPLOY_DIR`

| Scenario | Task                 | Provider | K8s Version      | V4_CFG_DEPLOYMENT_ASSETS | V4_CFG_LICENSE | V4_CFG_CERTS | Deployment Method |  Notes                                               |
|----------|----------------------|----------|------------------|--------------------------|----------------|--------------|-------------------|------------------------------------------------------|
| 1        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | UNSET                    | UNSET          | UNSET        | Docker            |  orders cli, assets, license, and certs downloaded   |
| 2        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | UNSET                    | UNSET          | SET          | Docker            |  orders cli, assets, and license  downloaded         |
| 3        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | UNSET                    | SET            | UNSET        | Docker            |  orders cli, assets, and certs downloaded            |
| 4        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | UNSET                    | SET            | SET          | Docker            |  orders cli and assets downloaded                    |
| 5        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | SET                      | UNSET          | UNSET        | Ansible           |  orders cli, license, and certs downloaded           |
| 6        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | SET                      | UNSET          | SET          | Ansible           |  orders cli and license downloaded                   |
| 7        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | SET                      | SET            | UNSET        | Ansible           |  orders cli and certs downloaded                     |
| 8        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | SET                      | SET            | SET          | Docker            |  Nothing downloaded user provided all required files |
| 9        | OOTB - DEPLOY: false | GCP      | v1.23.14-gke.401 | SET                      | SET            | SET          | Ansible           |  Nothing downloaded user provided all required files |

### CODE REVIEW

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -119,8 +119,9 @@ When V4_CFG_MANAGE_STORAGE is set to `true`, a new storage class is created: sas
...
 | V4_CFG_CADENCE_NAME | Cadence name | string | lts | false | [stable,lts] | viya |
 | V4_CFG_CADENCE_VERSION | Cadence version | string | "2022.09" | true | This value must be surrounded by quotation marks to accommodate the updated SAS Cadence Version format. If the value is not quoted the deployment will fail. | viya |
-| V4_CFG_DEPLOYMENT_ASSETS | Path to pre-downloaded deployment assets | string | | false | Leave blank to download deployment assets | viya |
-| V4_CFG_LICENSE | Path to pre-downloaded license file | string | | false| Leave blank to download license file | viya |
+| V4_CFG_DEPLOYMENT_ASSETS | Path to pre-downloaded deployment assets | string | | false | Leave blank to download [deployment assets](https://go.documentation.sas.com/doc/en/sasadmincdc/default/itopscon/n08bpieatgmfd8n192cnnbqc7m5c.htm#n1x7yoeafv23xan1gew0gfipt9e9) | viya |
+| V4_CFG_LICENSE | Path to pre-downloaded license file | string | | false| Leave blank to download the [license file](https://go.documentation.sas.com/doc/en/sasadmincdc/v_036/itopscon/n08bpieatgmfd8n192cnnbqc7m5c.htm#p1odbfo85cz4r5n1j2tzx9zz9sbi) | viya |
```
Comment (riragh) [inline docs/CONFIG-VARS.md:123]: Need to update "v_036" to default

File: docs/CONFIG-VARS.md [hunk 2]
```diff
@@ -119,8 +119,9 @@ When V4_CFG_MANAGE_STORAGE is set to `true`, a new storage class is created: sas
...
+| V4_CFG_CERTS | Path to pre-downloaded certificates file | string | | false| Leave blank to download the [certificates file](https://go.documentation.sas.com/doc/en/sasadmincdc/v_036/itopscon/n08bpieatgmfd8n192cnnbqc7m5c.htm#n0pj0ewyle0gfkn1psri3kw5ghha) | viya |
```
Comment (riragh) [inline docs/CONFIG-VARS.md:124]: same as above.

**Review by riragh — CHANGES_REQUESTED** [review]

File: docs/CONFIG-VARS.md (hunk 1 above)
Comment (jarpat) [reply docs/CONFIG-VARS.md]: Good catch, doc changes made.

**Review by riragh — APPROVED** [review]
LGTM

**Review by anthoday — APPROVED** [review]

**Review by dhoucgitter — APPROVED** [review]

**Review by thpang — APPROVED** [review]
LGTM

**Pull request merged**